File: pset1-c/mario/less/mario.c
// make mario.c
// check50 cs50/problems/2022/x/mario/less
// style50 mario.c

#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int n;
    do
    {
        n = get_int("Altura: ");
    }
    while (n > 8 || 1 > n);

    for (int i = 0; i < n; i++)
    {
        for (int j = 1; j < n - i; j++)
        {
            printf(" ");
        }

        //Hashes a esquerda
        for (int j = 0; j <= i; j++)
        {
            printf("#");
        }

        printf("\n");
    }
}

File: pset6-Python/mario/more/mario.py
# python mario.py
# check50 cs50/problems/2023/x/sentimental/mario/more
# style50 mario.py
# submit50 cs50/problems/2023/x/sentimental/mario/more

from cs50 import get_int

while True:
    n = get_int('Altura: ')
    if n < 1 or n > 8:
        n = get_int('Altura: ')
    if n >= 1 or n <= 8:
        break
for i in range(n):
    print((n - 1 - i) * " ", end="")
    print((i + 1) * "#" + "  " + (i + 1) * "#")